docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -176,33 +178,25 @@ fi
 
 TEST_PKG_IDENT="${1}"
 export TEST_PKG_IDENT
+
 hab pkg install core/bats --binlink
 hab pkg install "${TEST_PKG_IDENT}"
-bats "$(dirname "${0}")/test.bats"
+bats "${TESTDIR}/test.bats"
 ```
 
 BATS test script `7zip/tests/test.bats`:
 
 ```bash
-TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
-@test "7z exe runs" {
-  run hab pkg exec ${TEST_PKG_IDENT} 7z
-  [ $status -eq 0 ]
-}
-
-@test "7z version matches ${TEST_PKG_VERSION}" {
-  run hab pkg exec ${TEST_PKG_IDENT} 7z
-  [[ "$output" =~ 7-Zip\ \[64\]\ ${TEST_PKG_VERSION} ]]
-}
+expected_version="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
 
-@test "7za exe runs" {
-  run hab pkg exec ${TEST_PKG_IDENT} 7za
-  [ $status -eq 0 ]
+@test "7z version matches ${expected_version}" {
+   run hab pkg exec "${TEST_PKG_IDENT}" 7z
+   grep -Eo "7-Zip \[64\] $expected_version" <<< "${output}"
```

Comment:
I preferred the previous version of this, that showed the `@test` ending with a `[]` test.

What's currently here is 100% acceptable as a test, but it's maybe not exemplary?

Reply:
If the `run` tests get restored then this is maybe good as it is - we will have examples of how to do `[]` tests and also tests that just expect a command to exit 0

Reply:
Added back missing ``[ $status -eq 0 ]`` tests to illustrate both ways of testing the results